### DISCUSSION TITLE
How to change footer on minimal-blog template?

### DISCUSSION THREAD

**Discussion by tashasyarifah** [LekoArts/gatsby-themes#779] [Questions] [answered]
Hi, I want to change the footer. 

I already add `src/@lekoarts/components/footer.jsx` and edit them (such as try to add social media). However, nothing change. 

Did I missed something? Any help would be amazing.

Thank you!

**Comment by ch4nd4n** [comment, marked as answer]
I am using `src/@lekoarts/gatsby-theme-minimal-blog/components/footer.tsx` it works for me.
Source: https://github.com/ch4nd4n/chandankumar.com/blob/master/src/%40lekoarts/gatsby-theme-minimal-blog/components/footer.tsx

Its deployed on my site chandankumar.com

**Reply by tashasyarifah** [reply]
Thank you, I'll try it.
